k9mail/src/main/java/com/fsck/k9/view/MessageCryptoDisplayStatus.java

```diff
@@ -237,12 +252,15 @@ private static MessageCryptoDisplayStatus getDisplayStatusForPgpResult(CryptoRes
             }
         }
 
+        final long delta = signatureResult.getSignatureTimestamp().getTime() - messageSentDate.getTime();
```

Comment:
@Valodim I am not really sure in this case, which app should make the decision. Seems a protocol dependent check. Actually, we also need the same functionality for XMPP, so this would be an argument for including it in OpenKeychain.